dev/custom-pages/social.js

```diff
@@ -0,0 +1,34 @@
+(function() {
+    
```

Comment:
I don't think this is necessary. I added the sticky-top class to the div on the social page and it appears to be affixing the widget.